src/outcome_spaces/transfer_operator/transfer_operator.jl

```diff
@@ -2,12 +2,12 @@ using DelayEmbeddings, SparseArrays
 using StaticArrays
 using Random
 
-include("GroupSlices.jl")
+include("utils.jl")
 
 export
     TransferOperator, # the probabilities estimator
     InvariantMeasure, invariantmeasure,
-    transfermatrix
+    transfermatrix, transferoperator
```

Comment:
a new name `transferoperator` is exported as public but is not added in the docs.